.github/jobs/configure-checks/all.bats

```diff
@@ -0,0 +1,371 @@
+    assert_line "checking whether configure should try to set LDFLAGS... yes"
+    assert_line "checking whether the C compiler works... yes"
+    assert_line "checking for C compiler default output file name... a.out"
+    assert_line "checking for suffix of executables... "
+    assert_line "checking whether we are cross compiling... no"
+    assert_line "checking for suffix of object files... o"
+    assert_line "checking whether we are using the GNU C compiler... yes"
+    assert_line "checking whether C compiler accepts -Wall... yes"
+    assert_line "checking whether C compiler accepts -fstack-protector... yes"
+    assert_line "checking whether C compiler accepts -fPIE... yes"
+    assert_line "checking whether C compiler accepts -D_FORTIFY_SOURCE=2... yes"
+    assert_line "checking whether the linker accepts -fPIE... yes"
+    assert_line "checking whether the linker accepts -pie... yes"
+    assert_line "checking whether the linker accepts -Wl,-z,relro... yes"
+    assert_line "checking whether the linker accepts -Wl,-z,now... yes"
+    assert_line "checking whether $1 accepts -g... yes"
+    assert_line "checking for $1 option to accept ISO C89... none needed"
+    assert_line "checking whether $1 accepts -g... (cached) yes"
+    if [ -n "$2" ]; then
+        assert_line "checking whether $2 accepts -g... yes"
+        assert_line "checking how to run the C preprocessor... $1 -E"
+        assert_line "checking how to run the C++ preprocessor... $2 -E"
+    fi
+}
+
+compile_assertions_finished () {
+    assert_line " * CFLAGS..............: -g -O2 -Wall -fstack-protector -fPIE -D_FORTIFY_SOURCE=2"
+    assert_line " * CXXFLAGS............: -g -O2 -Wall -fstack-protector -fPIE -D_FORTIFY_SOURCE=2"
+    assert_line " * LDFLAGS.............:  -fPIE -pie -Wl,-z,relro -Wl,-z,now"
+}
+
+@test "Install GNU C only" {
+    repo-remove clang g++
+    repo-install gcc libcgroup-dev
+    compiler_assertions gcc ''
+    assert_line "checking for gcc... gcc"
+    assert_line "checking whether gcc accepts -g... yes"
+    assert_line "configure: error: C++ preprocessor \"/lib/cpp\" fails sanity check"
+}
+
+@test "Install GNU C++ only" {
+    # This does work due to dependencies
+    repo-remove clang gcc
+    repo-install g++ libcgroup-dev
+    compiler_assertions gcc g++
+    assert_line "checking for gcc... gcc"
+    assert_line "checking baseurl... checking for g++... g++"
+    compile_assertions_finished
+}
+
+@test "Install GNU C/C++ compilers" {
+    # The test above already does this
+    # repo-remove clang
+    # repo-install gcc g++ libcgroup-dev
+    # compiler_assertions
+    # compile_assertions_finished
+}
+
+@test "Install C/C++ compilers (Clang as alternative)" {
+    remove_gnuc
+    repo-install clang libcgroup-dev
+    compiler_assertions cc c++
+    assert_line "checking for gcc... no"
+    assert_line "checking for cc... cc"
+    assert_line "checking baseurl... checking for g++... no"
```

Comment:
This looks wrong: there should be output coming from the baseurl check.

Reply:
I agree it should, but when I run `make clean; make configure; ./configure | grep baseurl` this is the output I get in my testsetup.

Reply:
I've added a fix for this.